src/doc/building_visit/Advanced_Usage.rst

```diff
@@ -71,6 +71,22 @@ library is substituted at run time for OpenGL when running the parallel
 engine to enable scalable rendering. If you specify ``--mesagl`` then
 ``--osmesa`` is unnecessary and ignored if specified.
 
+
+Building Server only
+~~~~~~~~~~~~~~~~~~~~
+Much of the time, the pre-built VisIt executables will suffice for desktop systems but you need a special version of VisIt to run on a remote server and process data there.
+In this case, you can run client/server from the desktop to the remote server.
+This means that for the remote server build of VisIt, you need only build the server programs that will read and process the data since the GUI will run locally.
+You can speed up the VisIt build considerably by adding the ``--server-components-only`` flag to build only the VisIt server and related programs.
+
+.. code:: bash
+
+  env PAR_COMPILER=mpicc ./build_visit3_3_3 --server-components-only --mesa --icet
+
+This will do a basic build, but may not include the IO libraries you need.
+You can add ``--all-io`` in order to build all the IO libraries, or specify them individually, eg  ``--hdf5 --netcdf --conduit --mfem`` to add just the HDF5, NetCDF, Conduit and Mfem IO libraries.
```

Comment:
I'm not sure ``--all-io`` should be recommended. It should only be used by experts since the probability of one of the io library builds failing is high. Does ``--server-components-only`` turn off most of the io libraries? If not, then it should build HDF5, NetCDF, etc.

Reply:
I can remove that.
Only CMake, Python, VTK and ZLIB are built with --server-components-only.

Reply:
In that case, I would change the sentence to give the example of adding hdf5, netcdf, conduit and mfem. Maybe something like:

You can individually list the IO libraries you need, such as ``--hdf5 --netcdf --conduit --mfem``.


We should probably change the behavior of turning off all the IO libraries with ``--server-components-only``.